docs/core/docker/build-container.md

```diff
@@ -0,0 +1,391 @@
+
+
+```dockerfile
+COPY app\bin\Release\netcoreapp2.2\publish\ .\app
+
+ENTRYPOINT ["dotnet" "app/myapp.dll"]
+```
+
+The `COPY` command tells Docker to copy the specified folder on your computer to a folder in the container. In this example, the **publish** folder is copied to a folder named **app** in the container.
+
+The next command, `ENTRYPOINT`, tells docker to configure the container to run as an executable. After you start a container based on this image, it will automatically stop when the process identified from the `ENTRYPOINT` command ends.
+
+Save the file. From your terminal, run `docker build -t myimage .` and when that command finishes, run `docker images`.
+
+```console
+> docker build -t myimage .
+Sending build context to Docker daemon  819.7kB
+Step 1/3 : FROM mcr.microsoft.com/dotnet/core/runtime:2.2
+ ---> d51bb4452469
+Step 2/3 : COPY app/bin/Release/netcoreapp2.2/publish/ app/
+ ---> a1e98ac62017
+Step 3/3 : ENTRYPOINT ["dotnet", "app/myapp.dll"]
+ ---> Running in f34da5c18e7c
+Removing intermediate container f34da5c18e7c
+ ---> ddcc6646461b
+Successfully built ddcc6646461b
+Successfully tagged myimage:latest
+
+
+> docker images
+REPOSITORY                              TAG                 IMAGE ID            CREATED             SIZE
+myimage                                 latest              ddcc6646461b        10 seconds ago      314MB
+mcr.microsoft.com/dotnet/core/runtime   2.2                 d51bb4452469        2 days ago          314MB
+```
+
+Each command in the *Dockerfile* generated a layer and created an **IMAGE ID**. The final **IMAGE ID** (yours will be different) is **ddcc6646461b** and next you'll create a container based on this image.
+
+## Create a container
+
+Now that you have an image that contains your app, you can create a container. You can create a container in two ways. First, create a new container that is stopped.
+
+```console
+> docker create myimage
+0e8f3c2ca32ce773712a5cca38750f41259a4e54e04bdf0946087e230ad7066c
+```
+
+The `docker create` command from above will create a container based on the **myimage** image. The output of that command shows you the **CONTAINER ID** of the created container. To see a list of *all* containers, use the `docker ps -a` command:
+
+```console
+> docker ps -a
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS        PORTS   NAMES
+0e8f3c2ca32c        myimage             "dotnet app/myapp.dll"   4 seconds ago       Created               boring_matsumoto
+```
+
+### Manage the container
+
+Each container is assigned a random name that you can use to refer to that container instance. For example, the container that was created automatically chose the name **boring_matsumoto** and that name can be used to start the container. The following example uses the `docker start` command to start the container, and then uses the `docker ps` command to only show containers that are running:
```

Comment:
I'd mention that you can name a container when you create (or run) it. I wouldn't change the flow of this tutorial, but I think that would anticipate reader questions.